tests/test_web_exceptions.py

```diff
@@ -232,3 +232,16 @@ async def show(request):
     assert str(resp.url)[-5:] == "/show"
     text = await resp.text()
     assert text == "works"
+
+
+@pytest.mark.xfail(
+    raises=AttributeError,
+    reason="Regression in v3.7: https://github.com/aio-libs/aiohttp/issues/5657",
+)
+def test_body_unauthorized() -> None:
```

Comment:
```suggestion
def test_unicode_text_body_unauthorized() -> None:
```

Reply:
(the test function names should be descriptive enough to give the test report readers some idea of what this test is about)

Reply:
(also, it's usually a good idea to add a PEP 257 compliant docstring)